src/scss/patterns/_lists.scss

```diff
@@ -1,9 +1,5 @@
 @use '../config';
 
-/// # Typography
-/// Basic typographic styles for the site.
-/// @group type
-
```

Comment:
Not sure why, but using the node API caused SassDoc to pick up this file after the `type.scss` file, so this header wasn't at the top of the page.

Reply:
I'm guessing that has something to do with the change to `inDir` in `compile-docs`, and some odd globbing rule. Did it go from sorting by filename to pathname?